src/client/Client.cc

```diff
@@ -1495,6 +1495,13 @@ int Client::make_request(MetaRequest *request,
     if (request->aborted)
       break;
 
+    /* force quit if there's no MDS up */
+    if ((mdsmap->get_epoch() > 0) && (mdsmap->get_num_up_mds() == 0)) {
```

Comment:
get_num_mds() instead of get_num_up_mds(), when all mds are failed, we want client to wait for MDS to restart. Besides, I'd like to only apply the check for the requests from mount